### ISSUE TITLE
Ad-hoc attribute paths

### ISSUE THREAD

**Issue by danigulyas** [Cosium/spring-data-jpa-entity-graph#3]
Hey Guys and Gals!

So i've ran into the same problem as you did with Spring Data JPA as you did, i'm super happy that you made this plugin to solve it, huge-huge kudos, it already made my life easier!

I was thinking, Spring has [ad-hoc entity graphs](http://docs.spring.io/spring-data/data-jpa/docs/current/reference/html/#jpa.entity-graph) (see example 60th):

```java
@Repository
public interface GroupRepository extends CrudRepository<GroupInfo, String> {

  @EntityGraph(attributePaths = { "members" })
  GroupInfo getByGroupName(String name);

}
```

Would it be convenient to do this:
```java
final ProductRepository repo;
repo.findAll(EntityGraphUtils.fromAttributePaths("brand", "maker"));
```
?

I would be happy to research and make a pr about this if you think it would be helpful as well, let me know what do you think. :)

Have a pancake flavoured day,
-Dan

**Comment by reda-alaoui** [comment]
Hello Dan,

Thanks for the kind words :)

Yeah, I think it would be a great EntityGraph helper method !
I suppose that the method would return an instance of `DynamicEntityGraph`.

Feel free to open a PR ;)

**Comment by danigulyas** [comment]
Hi Réda!

Thanks for the fast response, i'll get to it in the evenings, hoping to make a PR in the coming days.

**Comment by reda-alaoui** [comment]
This will be available in release 1.10.14.
The release has been performed, will be available in the next hour.

**Comment by danigulyas** [comment]
Cool! Thx @reda-alaoui!
